lib/commons/aria/is-accessible-ref.js

```diff
@@ -1,13 +1,24 @@
 /* global aria, axe, dom */
-function findDomNode(node, functor) {
-	if (functor(node)) {
-		return node;
+const idRefsRegex = /^idrefs?$/;
+
+function cacheIdRefs(node, refAttrs) {
+	if (node.hasAttribute) {
+		if (node.nodeName.toUpperCase() === 'LABEL' && node.hasAttribute('for')) {
+			axe._cache.idRefs[node.getAttribute('for')] = 1;
```

Comment:
what happens if my `for` value is an `empty` space? ;)

`<label for=' '>Temp</label>`

Reply:
Then it would match any node with an `id` of an empty space I guess when checking idRefs. Though I don't see that as a problem

Reply:
I'm ok with this not being sanitized. It only adds the empty spaces `' '` to the cache, but that won't do anything unless an `id` matches the cache key exactly `<button id=" "></button>`, which is an invalid value for an id anyway.